downloadTimeCalculator.py

```diff
@@ -0,0 +1,95 @@
+# Write a procedure download_time which takes as inputs a file size, the
+# units that file size is given in, bandwidth and the units for
+# bandwidth (excluding per second) and returns the time taken to download
+# the file.
+# Your answer should be a string in the form
+# "<number> hours, <number> minutes, <number> seconds"
+
+# Some information you might find useful is the number of bits
+# in kilobits (kb), kilobytes (kB), megabits (Mb), megabytes (MB),
+# gigabits (Gb), gigabytes (GB) and terabits (Tb), terabytes (TB).
+
+# print 2 ** 10      # one kilobit, kb
+# print 2 ** 10 * 8  # one kilobyte, kB
+
+# print 2 ** 20      # one megabit, Mb
+# print 2 ** 20 * 8  # one megabyte, MB
+
+# print 2 ** 30      # one gigabit, Gb
+# print 2 ** 30 * 8  # one gigabyte, GB
+
+# print 2 ** 40      # one terabit, Tb
+# print 2 ** 40 * 8  # one terabyte, TB
+
+# Often bandwidth is given in megabits (Mb) per second whereas file size
+# is given in megabytes (MB).
+
+sizeDict = {'kb':2**10, 'kB':2**10*8, 'Mb': 2**20, 'MB':2**20*8, 'Gb': 2**30, 'GB':2**30*8, 'Tb': 2**40, 'TB': 2**40*8}
+
+
+def convert_seconds(number):
+    number1 = int(number)
+    decimal = round(number - number1, 10)
+    hour = int(number1 / 3600)
+    number1 = number1 - (hour * 3600)
```

Comment:
You can consider using a function `d, r = divmod(X, Y)`. It is equivalent to `d = int (X / Y)` and `r = X % Y`